packages/core/src/fonts.css

```diff
@@ -42,10 +38,8 @@
 	font-style: normal;
 	font-weight: 400;
 	src: local('Open Sans Regular'), local('OpenSans-Regular'),
-		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin.woff2)
-			format('woff2'),
-		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin.latin-ext.woff)
-			format('woff');
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin.woff2) format('woff2'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin.latin-ext.woff) format('woff');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Quote this URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aquoted-url)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this protocol-relative URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aprotocol-relative-url)